packages/material-ui-lab/src/Skeleton/Skeleton.test.js

```diff
@@ -20,6 +20,27 @@ describe('<Skeleton />', () => {
   });
 
   describeConformance(<Skeleton />, () => ({
+    classes,
+    inheritComponent: 'span',
+    mount,
+    refInstanceof: window.HTMLSpanElement,
+  }));
+
+  describeConformance(<Skeleton variant="text" />, () => ({
```

Comment:
Let's use a single `describeConformance`.

Reply:
so only the first one without a variant?
what is the error of the broken check? i don't get it

Reply:
One with the default props should great.

Reply:
We can have a small dedicated test instead.

Reply:
ok if i run `yarn docs:api` the docs will get updated. but than the default value is messed up

Reply:
What do you mean by "messed up", like more accurate? In this case, using span sounds better (which mean giving less weight to the div vs span semantic, as negligible: https://stackoverflow.com/questions/2961889/is-there-a-semantic-difference-spans-and-divs/2961920).